### ISSUE TITLE
HMDB51 overfitting

### ISSUE THREAD

**Issue by yongfali** [chenchao666/Video-Action-Recognition#1]
HMDB51 datasets are overfittting.How to solve this prolbems.

**Comment by chenchao666** [comment]
> HMDB51 datasets are overfittting.How to solve this prolbems.

What is the accuracy you get， your model needs to be pretrained in imagenet or Kinetics.
